packages/blade/docs/tokens/Colors.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, DocsContainer, Story, ColorPalette, ColorItem } from '@storybook/addon-docs';
+import { useTheme, BladeProvider } from '@razorpay/blade/components';
+import { paymentTheme, bankingTheme, colors } from '@razorpay/blade/tokens';
+import flat from 'flat';
+import get from 'lodash/get';
+
+<Meta
+  title="Tokens/Colors"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokens === 'paymentTheme') {
```

Comment:
It's not too obvious of a change when you change the theme or scheme. For a while I thought it's not working on my local until I scrolled down to see some of the colors changing. Can we put an indicator somewhere on the top which tells you which is the current selected theme and scheme?